python/lsst/sdm/schemas/dp1.yaml

```diff
@@ -6477,6 +6563,7 @@ tables:
     tap:column_index: 7
 - name: CcdVisit
   description: Defines a single detector of a visit
+  primaryKey: '#CcdVisit.ccdVisitId'
```

Comment:
This probably really isn't a useful primary key any more, because none of the other tables have ccdVisitId any more.  I think we have to use (visit, detector) which is on your other ticket.